tool/tsh/proxy.go

```diff
@@ -355,13 +355,23 @@ func onProxyCommandDB(cf *CLIConf) error {
 		}
 	}()
 
+	onNewConnection := func(dbCert *x509.Certificate, lp *alpnproxy.LocalProxy, conn net.Conn) {
+		if time.Now().After(dbCert.NotAfter) {
+			fmt.Fprintln(cf.Stdout())
+			fmt.Fprintln(cf.Stdout(), "Your database session is expired. Please restart the local proxy.")
```

Comment:
```suggestion
			fmt.Fprintln(cf.Stdout(), "Your database session has expired. Please restart the local proxy.")
```

I think "has" is more correct here though I'm not 100% sure.